lib/presenter.rb

```diff
@@ -5,20 +5,13 @@ def initialize(grid)
     @grid = grid
   end
 
-  def print
-    STDOUT.print "\n"
-    grid.height.times do |y|
-      grid.width.times do |x|
-        print_cell(grid.cells[y][x])
-      end
-      STDOUT.print "\n"
-    end
+  def print_grid
```

Comment:
Instead of printing out each character individually, only print fully-formed frames.